laravel-app/resources/views/profile/edit.blade.php

```diff
@@ -9,37 +9,67 @@
-                                    <div class="letter-bg">
-                                        {{ substr($user->full_name, 0, 1) }}
+                            @if (!$user->image_path)
+                                <div class="d-flex justify-content-center align-items-center">
+                                    <div class="profile-letter">
+                                        <div class="letter-bg">
+                                            {{ substr($user->full_name, 0, 1) }}
+                                        </div>
                                     </div>
                                 </div>
                             @else
-                                <img id="profileImage" class="img-fluid w-50 profile-image" src="{{ asset($user->image_path) }}" alt="Profile Picture">
+                                <img id="profileImage" class="img-fluid w-50 profile-image"
+                                    src="{{ asset($user->image_path) }}" alt="Profile Picture">
                             @endif
-                        </div>
-                    </div>
-                    <form method="POST" action="{{ route('profile.store') }}" enctype="multipart/form-data">
-                        @csrf
-                        <div class="row justify-content-center align-items-center">
-                            <div class="col-auto">
-                                <div class="form-group">
-                                    <label for="profile_image">
-                                        <a class="button button-secondary"><i class="fa-regular fa-image"></i></a>
-                                        <input type="file" class="form-control-file" id="profile_image" name="profile_image" hidden>
-                                    </label>
-                                </div>
-                            </div>
-                            <div class="col-auto">
-                                <button type="submit" class="button button-primary">Upload Image</button>
+                            <div class="d-flex align-items-center justify-content-center my-3">
+                                <form id="delete-profile-image" method="POST"
+                                    action="{{ route('profile.destroy', $user) }}">
+                                    @csrf
+                                    @method('DELETE')
+                                    <div class="col-auto">
+                                        <button type="submit" class="button button-danger deleteButton me-5"
+                                            onclick="confirmDelete()">
+                                            <i class="fa-regular fa-trash-can"></i>
+                                        </button>
+                                    </div>
+                                </form>
+                                <form method="POST" action="{{ route('profile.store') }}" enctype="multipart/form-data">
```

Comment:
Why still using html form action? I thought you guys using fetch or ajax for HTTP requests?

Reply:
We are using fetch and ajax in comment edit sir to improve the UI functionality, for the store and delete we opt to use html form action po.

Reply:
What?? but why? Paano kapag tinadtad nanaman namin yung submission niyan? hindi na ba nag ttime out?

Reply:
ohh that make sense sir, will resolve it 🙇